java/src/main/native/include/jni_utils.hpp

```diff
@@ -68,6 +69,32 @@ inline void check_java_exception(JNIEnv *const env) {
   }
 }
 
+/**
+ * @brief Helper to convert a pointer to a jlong.
+ *
+ * This is useful when, for instance, converting a cudf::column pointer
+ * to a jlong, for use in JNI.
+ */
+template <typename T> jlong to_jlong(T *ptr) {
```

Comment:
nit: I am not thrilled with the name. Could we call this `ptr_to_jlong` or `addr_to_jlong` so it is clear that this returns the address as a `jlong` and not the value in it?

Reply:
Hmm. Thanks for pointing that out, @revans2. That hadn't occurred to me. I'll make the change. 
`ptr_to_jlong()`, if that's acceptable?

Reply:
I am fine with that.

Reply:
Changed to `ptr_as_jlong()`. I would have preferred the shorter name, but it's best to avoid the ambiguity that @revans2 pointed out.